tst/EnergyPlus/unit/Coils/CoilCoolingDXCurveFitOperatingMode.unit.cc

```diff
@@ -209,3 +210,63 @@ TEST_F(CoilCoolingDXTest, CoilCoolingDXCurveFitOperatingMode_Sizing)
+         "  773.3,                                  !- Unit Internal Static Air Pressure {Pa}",
+         "  Discrete,                               !- Capacity Control Method",
+         "  0,                                      !- Evaporative Condenser Basin Heater Capacity {W/K}",
+         "  2,                                      !- Evaporative Condenser Basin Heater Setpoint Temperature {C}",
+         "  ,                                       !- Evaporative Condenser Basin Heater Operating Schedule Name",
+         "  Electricity,                            !- Compressor Fuel Type",
+         "  Coil Cooling DX Curve Fit Operating Mode 1, !- Base Operating Mode",
+         ",",
+         ",",
+         "heaterCapCurve;          !- Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name",
+
+         "Curve:Linear,",
+         "heaterCapCurve,          !- Name",
+         "10.0,                    !- Coefficient1 Constant",
+         "2.,                      !- Coefficient2 x",
+         "10.0,                    !- Minimum Value of x",
+         "70;                      !- Maximum Value of x",
+
+         "Coil:Cooling:DX:CurveFit:OperatingMode,",
+         "  Coil Cooling DX Curve Fit Operating Mode 1, !- Name",
+         "  Autosize,                               !- Rated Gross Total Cooling Capacity {W}",
+         "  Autosize,                                    !- Rated Evaporator Air Flow Rate {m3/s}",
+         "  Autosize,                               !- Rated Condenser Air Flow Rate {m3/s}",
+         "  0,                                      !- Maximum Cycling Rate {cycles/hr}",
+         "  0,                                      !- Ratio of Initial Moisture Evaporation Rate and Steady State Latent Capacity {dimensionless}",
+         "  0,                                      !- Latent Capacity Time Constant {s}",
+         "  0,                                      !- Nominal Time for Condensate Removal to Begin {s}",
+         "  No,                                     !- Apply Latent Degradation to Speeds Greater than 1",
+         "  EvaporativelyCooled,                    !- Condenser Type",
+         "  Autosize,                               !- Nominal Evaporative Condenser Pump Power {W}",
+         "  1,                                      !- Nominal Speed Number",
+         "  Coil Cooling DX Curve Fit Speed 1;      !- Speed Name 1"});
+
+    idf_objects += this->getSpeedObjectString("Coil Cooling DX Curve Fit Speed 1");
+    EXPECT_TRUE(process_idf(idf_objects, false));
+    int coilIndex = CoilCoolingDX::factory(*state, "Coil Cooling DX 1");
+    auto const &thisCoil(state->dataCoilCooingDX->coilCoolingDXs[coilIndex]);
+    EXPECT_EQ("COIL COOLING DX 1", thisCoil.name);
+    EXPECT_EQ("COIL COOLING DX CURVE FIT PERFORMANCE 1", thisCoil.performance.name);
+    EXPECT_EQ("HEATERCAPCURVE", Curve::GetCurveName(*state, thisCoil.performance.crankcaseHeaterCapacityCurveIndex));
+}
```

Comment:
Add a return to the last line.

Reply:
an explicit return like this `return;`? @mjwitte

Reply:
Sorry, no, a <cr> return. Look at the display in the PR and you'll see a red circle at the end, because the last line of the file is not terminated fully.

Reply:
Oh! I understand now. I'll add a new line in the end.